core/cat/looking_glass/stray_cat.py

```diff
@@ -17,7 +18,7 @@
 from cat.convo.messages import CatMessage, UserMessage, MessageWhy, Role
 
 MSG_TYPES = Literal["notification", "chat", "error", "chat_token"]
-
+MEMORY_COLLECTION = Literal["episodic", "declarative", "procedural"]
```

Comment:
No need for this, when you ask for core collections:

```python
collections = self.memory.vectors.collections.keys()
```

You can then loop over them and see if the requested ones are present.
If collection is not present, ignore and throw a warning